### ISSUE TITLE
Crash Report

### ISSUE THREAD

**Issue by kylongmu** [mu-editor/crash-reports#52]
**REMEMBER TO ATTACH YOUR LOG FILE**

Version: 1.1.0.beta.6
Locale: zh_CN
Platform: Windows 10 10.0.19043 AMD64

```
Traceback (most recent call last):
  File "C:\Users\THINKPAD\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\app.py", line 159, in run
    raise ex
  File "C:\Users\THINKPAD\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\app.py", line 146, in run
    venv.ensure_and_create(self.display_text)
  File "C:\Users\THINKPAD\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\virtual_environment.py", line 603, in ensure_and_create
    self.create()
  File "C:\Users\THINKPAD\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\virtual_environment.py", line 767, in create
    self.create_venv()
  File "C:\Users\THINKPAD\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\virtual_environment.py", line 798, in create_venv
    raise VirtualEnvironmentCreateError(
mu.virtual_environment.VirtualEnvironmentCreateError: Unable to create a virtual environment using C:\Users\THINKPAD\AppData\Local\Programs\Mu Editor\Python\pythonw.exe at C:\Users\THINKPAD\AppData\Local\python\mu\mu_venv-38-20211022-000243
ModuleNotFoundError: No module named 'virtualenv.seed.embed.via_app_data'

```

**Comment by carlosperate** [comment]
Hi @kylongmu 

I believe this should have been fixed in the latest Mu v1.1.1 release, could you try the latest version from https://codewith.mu/en/download and see if that works in your environment?

Thanks!

**Comment by kylongmu** [comment]
> Hi @kylongmu
> 
> I believe this should have been fixed in the latest Mu v1.1.1 release, could you try the latest version from https://codewith.mu/en/download and see if that works in your environment?
> 
> Thanks!

This version runs ok, this problem is fixed!
